### PR TITLE
Gke postgres scenario

### LINKED ISSUE #369: Scenario for GKE data services with PostgreSQL Hyperscale
Add new scenario to cover GKE PostgreSQL Hyperscale

### PR DESCRIPTION
**Pull request by dkirby-ms**
Closes #369

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_data/gke/gke_mssql_mi_terraform/_index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 type: docs
 title: "SQL Managed Instance Terraform plan"
 linkTitle: "SQL Managed Instance Terraform plan"
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_data/gke/gke_mssql_mi_terraform/_index.md:4]: 1. Change to Postgres
2. Change filename

**Review by likamrat — CHANGES_REQUESTED** [review]
1. 19.png - Any specific reason for having the first line selected?
2. 34+38.png - Align the data controller endpoint IP address

**Comment by dkirby-ms** [comment]
Fixed 19.png.

I dont see what you mean about 34-38. Looks like same IP to me?

**Review by likamrat — APPROVED** [review]
LGTM

**Pull request merged**